src/Redux/Actions/testReduxActions.js

```diff
@@ -0,0 +1,19 @@
+import types from ".";
+import axios from "../../axios";
+
+const { GET_WELCOME } = types;
+
+/** 
+ * example of a reducer
+ * you can export it as an object of functions(actions)
+ * or you can export one by one and access them in your component
+ * 
+ * */ 
+export default {
+  getWelcome: () => dispatch => {
+    axios
+      .get("/")
+      // I dispact the action passwing data as payload
```

Comment:
@alainburindi grammar correction here: `I dispatch the action passing data as payload`